helm/helm.go

```diff
@@ -198,6 +198,11 @@ func UpgradeDeployment(deploymentName, chartName string, values []byte, reuseVal
 
 //CreateDeployment creates a Helm deployment
```

Comment:
Maybe update the doc to say this function only creates a Deployment in a "default" namespace